src/containers/LoginContainer.jsx

```diff
@@ -13,6 +13,7 @@ import { useNavigate } from 'react-router-dom';
 import Loading from '../components/Loading';
 import { useDispatch } from 'react-redux';
 import { getToken } from '../states/features/login/actions';
+import { API_URL } from '../constants';
```

Comment:
Why is it comming from the constants though, why not have it in the `.env` file?

Reply:
We imported it from the constants to allow ourselves to be change it from one place and update in the rest of the code. [This is where we loaded it from the .env](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/bug-update-variables/src/constants/API.jsx#L1). 

```js
const API_URL = import.meta.env.VITE_API_URL;

export { API_URL }
```